### ISSUE TITLE
Ability to quickly play a feed (play last video and queue next ones)

### ISSUE THREAD

**Issue by luckkmaxx** [libre-tube/LibreTube#4554]
### Describe your suggested feature

I thought 2 ways in the UI:
1. A FAB play icon who will play and queue the selected feed group.
2. By long tapping a group show a "play all".

### Other details

this could be enhanced if it takes only the last non-watched or half-watched videos and starts to play it from older to newer

Just a crazy idea. Maybe I should have to open it on discussions.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I think it's an okay-ish idea, but I'm not sure how we should implement that visually - I think a FAB is probably an overkill and gives it too much importance, I don't think it's useful for a large part of users.

**Comment by luckkmaxx** [comment]
Yes, FAB is usually annoying for me too. Not this case for me as it will be always used for that meant behavior.
Maybe in some moment I find another place to this, but at the moment these 2 are in my mind.
Also could be optional places, where some people can activate the fab as secondary option, and by long pressing the groups as default option (maybe helps a hint like "long press to play all" when you tap a group for first time).

Then we also could have the option to choose: 
Play All > Older to newer
-------------> Newer to older

**Comment by luckkmaxx** [comment]
OK this is great! Thanks.
I was thinking another option also regarding groups edit way.
When you are in a group, the button is wrapped with color, and appears a checkmark wich confirms you are inside that group. This checkmark IMO is useless since you are wrapping the button with the theme color. Instead, this could be a good place to show a Play icon.
In the other hand, group editting could be shown when you long tap any group.
